vcf-dumper/vcf-dumper-ws/src/main/resources/application.properties

```diff
@@ -5,9 +5,6 @@
 #spring.datasource.password=
 #spring.datasource.driver-class-name=org.postgresql.Driver
 
-# EVAPRO JNDI configuration
-# spring.datasource.jndi-name =  java:/comp/env/jdbc/@eva.evapro.datasource@
-
 #Avoid hibernate ddl schema generation/update or validation
```

Comment:
Can we drop these 2 properties too? They are specific to a relational database that we are not using.